tasks/apply-mapping/README.md

```diff
@@ -25,6 +25,15 @@ This task supports variable expansion in tag values from the mapping. The curren
 | dataPath | Path to the JSON string of the merged data to use in the data workspace | No | |
 | failOnEmptyResult | Fail the task if the resulting snapshot contains zero components | Yes | false |
 
+## Changes in 1.5.0
+* Repository Handling:
+  * Added support for converting quay.io repository URLs to registry.redhat.io format and vice versa.
+  * Introduced a new key rh-registry-repo to store the updated repository value in the registry.redhat.io format.
+  * Added a new key registry-access-repo to store the repository value in the registry.access.redhat.com format.
+  * The original repository key now stores the repository value in the quay.io format to ensure backward compatibility.
+* Testing Enhancements:
```

Comment:
This isn't really necessary. We don't document unit test changes in the changelogs

Reply:
Fixed!